docs/_content/standaard/verzoeken/index.md

```diff
@@ -0,0 +1,46 @@
+
+Status van verzoeken: is in feite de status van de intake: nieuw, ingetrokken, in intake/behandeling, in behandeling genomen, afgewezen. Nadat een verzoek in behandeling is genomen (of afgewezen) kan het niet meer worden gewijzigd.
+
+# Interactie met de klant
+Terugkoppeling naar de klant over voortgang gebeurt in termen van de opgestarte zaken of andere activiteiten. Een verzoek heeft alleen een status m.b.t. voortgang van de intake die overigens ook in de PIP getoond kan worden (let op: dit kan verwarrend zijn als ook statussen van zaken getoond worden; dit is aan de consumers om op te lossen). De klant kan een overzicht van ingediende verzoeken opvragen en moet een verzoek kunnen intrekken. Een verzoek tot intrekken wordt gerelateerd aan het oorspronkelijke verzoek. Ook kan een klant aanvullingen doen op een verzoek. Dat leidt tot een nieuw verzoek dat aan het oorspronkelijke verzoek wordt gekoppeld.
+
+Een verzoek kan meerdere producten/diensten omvatten hetgeen een winkelmandje mogelijk maakt op de website van de gemeente. Als gevolg hiervan kan een verzoek tot meerdere zaken leiden. 
+
+# Gegevens: generiek en specifiek
+Bij een verzoek wordt alleen generieke metadata opgeslagen. Procesgegevens (domeinspecifiek) worden in een andere registratie opgeslagen. Bijv. bij een Melding Openbare Ruimte worden specifieke gegevens zoals locatie en meldinggegevens in een meldingen-registratie opgeslagen. Vanuit het verzoek wordt verwezen met een URI naar de melding.
+
+Wat betreft behandeltermijn (uiterste en geplande) volgen we de zaaktypen. Terzijde: deze zouden eigenlijk bij de producten in de PDC moeten staan, omdat producten de interface vormen tussen klant en dienstverlener. Voor nu volgen we het huidige model.
+
+# Typen verzoeken
+Er zijn verschillende typen verzoeken. Minimaal de volgende typen worden onderscheiden:
+* aanvragen product of dienst (uit de PDC van de gemeente)
+* doen van een melding (bijv. een melding openbare ruimte)
+* aanleveren gegevens bij een reeds bestaand verzoek
+* bezwaar maken (in feite geen verzoek om een product), maar wel een verzoek
+
+# Relaties/cardinaliteiten
+* Een contactmoment kan over een verzoek of een zaak gaan. Met een contactmoment kan geen verzoek worden ingediend. Meerdere contactmomenten kunnen aan een verzoek worden gekoppeld. 1:n van verzoek naar contacten.
+* De relatie tussen zaken en verzoeken is m:n. Een verzoek kan tot meerdere zaken leiden en een zaak kan gekoppeld zijn aan meerdere verzoeken (zie eerder voorbeeld over MOR).
+* Een verzoek kan meerdere producten omvatten. 1:n van verzoek naar producten
+* Een verzoek kan betrekking hebben op meerdere andere verzoeken. Bijv. het verzoek tot intrekken heeft betrekking op een of meer andere verzoeken. Het verzoek met aanvullende gegevens heeft betrekking op een ander verzoek. 1:n van verzoek naar verzoeken.
+* Een verzoek kan bijlagen hebben in de vorm van informatieobjecten. 1:n
+
```

Comment:
Wellicht toevoegen:

'Relaties tussen verzoeken en Zaken worden aan de Verzoek zijde vastgelegd. Een Zaak heeft dus geen weet van het verzoek waaraan het gerelateerd is.'